scripts/modal.js

```diff
@@ -1,3 +1,133 @@
+let htmlElements = '';
+
+const projects = [
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+];
+
+for (let i = 0; i <= 6; i += 1) {
+  htmlElements
+   += ` 
+   <div class="modal" id="modal2" data-animation="mixInAnimations">
+   <div class="modal-dialog">
+     <header class="modal-header">
+       <button class="close-modal" aria-label="close modal" data-close>
+         ✕  
+       </button>
+     </header>
+     <section class="modal-content">
```

Comment:
- [ ] Kindly add the right margin and padding  so it looks like the original design